CHANGES.md

```diff
@@ -3,6 +3,7 @@ Change Log
 
 ### 1.26 - 2016-10-03
 
+* Removed an unnecessary reprojection of Web Mercator imagery tiles to the Geographic projection on load.  This should improve both visual quality and performance slightly.
```

Comment:
Are you sure we can make the performance claim in practice?  In the steady state, this is probably worse.